File: python-sdk/src/planetscaleapi/models/operations/cancel_or_deny_a_demotion_request.py
import dataclasses



@dataclasses.dataclass
class CancelOrDenyADemotionRequestPathParams:
    database: str = dataclasses.field(metadata={'path_param': { 'field_name': 'database', 'style': 'simple', 'explode': False }})
    name: str = dataclasses.field(metadata={'path_param': { 'field_name': 'name', 'style': 'simple', 'explode': False }})
    organization: str = dataclasses.field(metadata={'path_param': { 'field_name': 'organization', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class CancelOrDenyADemotionRequestRequest:
    path_params: CancelOrDenyADemotionRequestPathParams = dataclasses.field()
    

@dataclasses.dataclass
class CancelOrDenyADemotionRequestResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    

File: go-sdk/pkg/models/operations/cancelordenyademotionrequest.go
package operations

type CancelOrDenyADemotionRequestPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Name         string `pathParam:"style=simple,explode=false,name=name"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type CancelOrDenyADemotionRequestRequest struct {
	PathParams CancelOrDenyADemotionRequestPathParams
}

type CancelOrDenyADemotionRequestResponse struct {
	ContentType string
	StatusCode  int
}
